internal/controllers/clusterextension_controller.go

```diff
@@ -101,8 +101,8 @@ func (r *ClusterExtensionReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	l := log.FromContext(ctx).WithName("operator-controller")
 	ctx = log.IntoContext(ctx, l)
 
-	l.V(1).Info("reconcile starting")
-	defer l.V(1).Info("reconcile ending")
+	l.Info("reconcile starting")
+	defer l.Info("reconcile ending")
```

Comment:
With the original zap logger `l.V(1).Info` was being output by default.

With klogr, we need to put these log messages at a lower verbosity level to keep them coming by default.